google/api_core/client_options.py

```diff
@@ -62,6 +62,9 @@ class ClientOptions(object):
             callback which returns client certificate file path, encrypted private
             key file path, and the passphrase bytes.``client_cert_source`` and
             ``client_encrypted_cert_source`` are mutually exclusive.
+        quota_project (str): An optional project name that a client's
+            quota belongs to.
+        credentials_file (str): An optional path to a file storing credentials.
```

Comment:
(nit)
Optional arguments are usually annotated as `Optional[str]` (at least in PubSub and BigQuery libs that I know).

Reply:
+1 We should probably update the args to use this style

Reply:
Done